unlock-app/src/components/interface/user-account/PaymentDetails.tsx

```diff
@@ -0,0 +1,78 @@
+/* eslint react/display-name: 0 */
+import React from 'react'
+import { connect } from 'react-redux'
+import {
+  ReactStripeElements,
+  StripeProvider,
+  Elements,
+  CardElement,
+  injectStripe,
+} from 'react-stripe-elements'
+import { SectionHeader, Column, Grid, SubmitButton } from './styles'
+import { signPaymentData } from '../../../actions/user'
+
+interface PaymentDetailsProps {
+  stripe: any
+  signPaymentData: (stripeTokenId: string) => any
+}
+
+export const PaymentDetails = React.memo(
```

Comment:
can you add a comment explaining why this needs to be memoized?

Reply:
Inbound